invenio_theme/templates/semantic-ui/invenio_theme/header.html

```diff
@@ -42,7 +42,7 @@
                     <div class="ui action icon input">
                       <input type="text" name="q"
                              placeholder="{{ _('Search') }}">
-                      <button type="submit" class="ui icon button orange invenio-theme-search-button">
+                      <button type="submit" class="ui icon button invenio-theme-search-button">
```

Comment:
Should ``search`` not be part of the CSS classes?

Reply:
Yes, I missed that :)